templates/takeovers/templates.html

```diff
@@ -88,11 +88,7 @@ <h3>Details</h3>
 {% include "takeovers/_template.html" %}
 {% endwith %}
 
-<section class="p-strip is-deep is-bordered">
```

Comment:
I removed the blog lists from this page because they seem unnecessary, and it's tricky to make multiple blog lists work on the same page with JavaScript & includes - 'cos we'd have to have unique IDs for the elements, which is troublesome and not worth solving.